src/frontend/components/navbar/nav.js

```diff
@@ -0,0 +1,47 @@
+import "./nav.css";
+import { useSidebar } from "../../contexts/index";
+import { useTheme } from "../../contexts/index";
```

Comment:
suggestion: both imports have the same path so you can import it from one.

Reply:
Yeah right  thanks for pointing out I will do it